cpp/src/link_analysis/pagerank.cu

```diff
@@ -350,6 +351,13 @@ void pagerank(experimental::GraphCSCView<VT, ET, WT> const &graph,
 {
   CUGRAPH_EXPECTS(pagerank != nullptr, "Invalid API parameter: Pagerank array should be of size V");
 
+  if (handle.comms_initialized()) {
```

Comment:
Is this a debugging artifact?
@afender do you need the pagerank code to be a no-op when it is in a multi gpu environment?
And if so, do you need some sort of a print in that case?

Reply:
This is a place holder for updates in #947 . The prints are going to be replaced by opg code

Reply:
That's ok for this PR, I have the upgrade in #947 that will replace this function.